File: 2025-10/2025-10-21/thermostat_adjuster_2.py
def adjust_thermostat(current_f, target_c):
    difference = round(target_c * 1.8 + 32 - current_f, 1)

    return f"Heat: {difference} degrees Fahrenheit" if difference > 0 \
        else f"Cool: {abs(difference)} degrees Fahrenheit" if difference < 0 \
        else "Hold"

print(adjust_thermostat(32, 0))
print(adjust_thermostat(70, 25))
print(adjust_thermostat(72, 18))
print(adjust_thermostat(212, 100))
print(adjust_thermostat(59, 22))

File: 2025-10/2025-10-21/thermostatAdjuster2.js
function adjustThermostat(currentF, targetC) {
  const difference = parseFloat(targetC * 1.8 + 32 - currentF).toFixed(1);

  return difference > 0 ? `Heat: ${difference} degrees Fahrenheit` 
    : difference < 0 ? `Cool: ${Math.abs(difference)} degrees Fahrenheit` 
    : "Hold";
}

console.log(adjustThermostat(32, 0));
console.log(adjustThermostat(70, 25));
console.log(adjustThermostat(72, 18));
console.log(adjustThermostat(212, 100));
console.log(adjustThermostat(59, 22));